tests/vmi_lifecycle_test.go

```diff
@@ -1205,11 +1205,17 @@ var _ = Describe("[rfe_id:273][crit:high][arm64][vendor:cnv-qe@redhat.com][level
 
 				By("adding a node-feature-discovery CPU model label to a node")
 				vmi := tests.NewRandomVMIWithEphemeralDiskAndUserdata(cd.ContainerDiskFor(cd.ContainerDiskCirros), "#!/bin/bash\necho 'hello'\n")
+				featureToRequire := supportedFeatures[0]
+
+				if featureToRequire == "fpu" {
+					featureToRequire = supportedFeatures[1]
+				}
+
 				vmi.Spec.Domain.CPU = &v1.CPU{
 					Cores: 1,
 					Features: []v1.CPUFeature{
 						{
-							Name:   supportedFeatures[0],
+							Name:   featureToRequire,
 							Policy: "require",
 						},
 						{
```

Comment:
To me this is a bit hard to understand. I would suggest something like:
```go
const featureToDisable = "fpu"

featureToRequire := supportedFeatures[0]

if featureToRequire == featureToDisable {
	// Picking another feature since this one is going to be disabled
	featureToRequire = supportedFeatures[1]
}

// ....

{
	Name:   featureToDisable,
	Policy: "disable",
},

```

Reply:
I agree with @iholder-redhat , but I would move all that code into an helper.

Reply:
Done.